airflow/models/taskinstance.py

```diff
@@ -181,6 +181,7 @@ class TaskInstance(Base, LoggingMixin):
     )
 
     def __init__(self, task, execution_date: datetime, state: Optional[str] = None):
+        super().__init__()
```

Comment:
I am not sure. Will it initialize both Base and Mixin :)? Just checking.

Reply:
As far as I understand - no -> https://stackoverflow.com/a/16310777/9713409
But it's not a problem here because we nearly never initialize the `LoggingMixin`, I think the constructor in it is part of backward compatibility